ReSharper.FSharp/src/FSharp.Common/src/Util/FSharpSymbolUtil.fs

```diff
@@ -146,8 +146,8 @@ let patternName (pattern: FSharpActivePatternGroup) =
     let wildCase = if pattern.IsTotal then "|" else "_|"
     "|" + joinedNames + wildCase
     
-[<Extension; CompiledName("GetAbbreviatedType")>]
-let getAbbreviatedType (entity: FSharpEntity) =
+[<Extension; CompiledName("GetAbbreviatedEntity")>]
+let getAbbreviatedEntity (entity: FSharpEntity) =
```

Comment:
I've added this extension before but GetAbbreviatedType is kinda wrong.. GetAbbreviatedEntity is a right one I believe